third_party/txt/tests/platform_mac_tests.cc

```diff
@@ -21,13 +21,14 @@ class PlatformMacTests : public ::testing::Test {
 TEST_F(PlatformMacTests, RegisterSystemFonts) {
   DynamicFontManager dynamic_font_manager;
   RegisterSystemFonts(dynamic_font_manager);
-  ASSERT_EQ(dynamic_font_manager.font_provider().GetFamilyCount(), 2ul);
+  ASSERT_EQ(dynamic_font_manager.font_provider().GetFamilyCount(), 1ul);
   ASSERT_NE(dynamic_font_manager.font_provider().MatchFamily(
                 "CupertinoSystemDisplay"),
             nullptr);
-  ASSERT_NE(
-      dynamic_font_manager.font_provider().MatchFamily("CupertinoSystemText"),
-      nullptr);
+  ASSERT_EQ(dynamic_font_manager.font_provider()
+                .MatchFamily("CupertinoSystemDisplay")
+                ->count(),
+            9);
```

Comment:
When I run this locally I'm seeing a size of 10 instead of 9
(8 values from 0 through 700, plus the extra 780 and 810 weights)